src/core/lib/view-state.js

```diff
@@ -28,8 +29,12 @@ export default class ViewState {
     return this._formatString(opts);
   }
 
+  getProps() {
+    return Object.assign({}, this._getOrbitProps(), this.props);
```

Comment:
This will add fields `pitch`, `bearing` and `postition` to the returned viewport props, which are not used by OrbitViewport.

Reply:
> This will add fields pitch, bearing and postition to the returned viewport props, which are not used by OrbitViewport.

Does it matter? They are all mathematical equivalents.  Ultimately I'd like OrbitViewport internally to use math.gl classes to convert the standard viewState methods rather than relying on extra methods.

Reply:
Yes, because these are sent back to the user via the `onViewportChange` callback. I'd be fine if the controllers have a method to extract the real props from this.